Explorer/Assets/DCL/InWorldCamera/InWorldCamera/UI/Animations/Active.anim

```diff
@@ -0,0 +1,359 @@
+%YAML 1.1
```

Comment:
I don't know how complex this animations are, but can't they be done in DOTween to avoid the anims?